### ISSUE TITLE
Tile Selection Box wrong size

### ISSUE THREAD

**Issue by 01010111** [Dovyski/flixel-studio#8]
It seems like the tile selection box is not accounting for zoom level correctly, making it difficult to select the correct tiles:
![box1](https://user-images.githubusercontent.com/1042453/46901141-ebf80780-ce7b-11e8-8a82-de1a9e81de3c.jpg)

**Comment by Dovyski** [comment]
Sorry about that, I'll look into it (and other zoom related issues) soon. After you select the tile you want, does the tile tool work on a maximized window?

**Comment by 01010111** [comment]
Yeah, outside of the tile selection box the tool works as expected!

**Comment by Dovyski** [comment]
Ok, thanks! I think the tile/zoom problem has been fixed in b0a9c41f32d53465c95300ad60c0c1f2f24cb4e3. Could you check if it solved your problem, please?

**Comment by 01010111** [comment]
The new version fixed the tile selection zoom issues, but now it seems that the cursor is offset:
![screen shot 2018-10-16 at 12 51 56 pm](https://user-images.githubusercontent.com/1042453/47033291-6b1e6180-d142-11e8-83b4-c88ab7c866ea.png)
This is on OSX, I can try at home on my Windows machine to see if it's Mac specific.

**Comment by 01010111** [comment]
to confirm, when I try to match the aspect ratio on a resized window things are working fine:
![screen shot 2018-10-16 at 12 55 22 pm](https://user-images.githubusercontent.com/1042453/47033447-d6683380-d142-11e8-92e3-20e58ed0499e.png)

**Comment by Dovyski** [comment]
Great! The weird cursor offset in a maximized window is an old problem (that I never looked into). Even though it is not an exclusive problem of the tile tool, I will keep this issue open so I can work on that.
